build-lab.sh

```diff
@@ -27,6 +27,10 @@ sudo apt-get install -y tree
 
 echo -e "${LINE_ASCII_CONSOLE}\n"
 
+echo "gem: --no-ri --no-rdoc" >> ${lab}/.gemrc
```

Comment:
https://stackoverflow.com/a/57475843/645647

are we still using an older version or should we be using this new option instead (--no-document)?